packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -45,8 +45,8 @@ data class GutenbergProps @JvmOverloads constructor(
         putBoolean(PROP_CAPABILITIES_MEDIAFILES_COLLECTION_BLOCK, enableMediaFilesCollectionBlocks)
         putBoolean(PROP_CAPABILITIES_UNSUPPORTED_BLOCK_EDITOR, enableUnsupportedBlockEditor)
         putBoolean(PROP_CAPABILITIES_CAN_ENABLE_UNSUPPORTED_BLOCK_EDITOR, canEnableUnsupportedBlockEditor)
-        putBoolean(PROP_CAPABILITIES_AUDIO_BLOCK, enableAudioBlock)
         putBoolean(PROP_CAPABILITIES_REUSABLE_BLOCK, enableReusableBlock)
+        putBoolean(PROP_CAPABILITIES_IS_AUDIO_BLOCK_MEDIA_UPLOAD_ENABLED, isAudioBlockMediaUploadEnabled)
```

Comment:
Nitpick: I saw that these lines match the same order as the function arguments but this line is after `PROP_CAPABILITIES_AUDIO_BLOCK`, I wouldn't block at all the PR because of this but I wanted to let you know in case you made other modifications and you would like to update this file too 😄 .

_NOTE: This also happens in the line `79`._

Reply:
This is good for readability so I am quite fine with making these changes 🍻 Made the changes and pushed them.